### PR TITLE
Completed virus class

### PR DESCRIPTION
**Pull request by bmoran17**
- Created virus class 
- Added properties and attributes of the virus used in simulation: name, reproduction rate, mortality rate

### CODE REVIEW

File: src/utils/virus.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class Virus {
+  constructor(name, repro_rate, mortality_rate) {
+    this.name = name;
+    this.repro_rate = repro_rate;
+    this.mortality_rate = mortality_rate;
+  }
+
+}
```
Comment (sprajjwal) [inline src/utils/virus.js]: need newline at end of file.

Comment (sprajjwal) [inline src/utils/virus.js]: class needs to be exported

Comment (bmoran17) [reply src/utils/virus.js]: Added.

Comment (bmoran17) [reply src/utils/virus.js]: Done.

**Review by sprajjwal — APPROVED** [review]
LGTM

**Pull request merged**